src/css/profile/mobile/common/checkbox.less

```diff
@@ -11,92 +11,40 @@ input[type="checkbox"].ui-checkbox:not(.ui-toggle-switch) {
 
 input[type="checkbox"].ui-checkbox:not(.favorite) {
```

Comment:
if you remove favourite support this should be also removed
:not(.favorite)

Reply:
Removed.